crmsh/corosync_config_format.py

```diff
@@ -8,12 +8,13 @@
 
 
 logger = logging.getLogger(__name__)
+COMMENT_PREFIX = '__comment'
```

Comment:
```suggestion
COMMENT_PREFIX = '#comment'
```

Using `#` will make sure it does not conflict with anything.